cmd/cluster-agent/admissioncontroller/listener.go

```diff
@@ -0,0 +1,191 @@
+package admissioncontroller
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"k8s.io/api/admission/v1beta1"
+	admissionregistrationv1beta1 "k8s.io/api/admissionregistration/v1beta1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
+	"k8s.io/kubernetes/pkg/apis/core/v1"
+
+	"github.com/DataDog/datadog-agent/cmd/agent/common/jsonpatch"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+)
+
+var (
+	// Scheme is the schemed used for validation and mutation of resources
```

Comment:
Because in Go the standard for documentation is to start each documentation snippet as a comment above the identifier, having the first word of the sentence being the identifier itself, and because there is a difference in Go between capital letter (unexported/private) and capital letter (exported/public) we'll have to lower case these.
```suggestion
	// scheme is the schemed used for validation and mutation of resources
```